tests/src/test/java/alluxio/client/cli/fs/command/CpCommandIntegrationTest.java

```diff
@@ -138,6 +161,86 @@ public void copyWildcard() throws Exception {
         equals(new AlluxioURI("/copy/foobar3"), new AlluxioURI(testDir + "/bar/foobar3")));
   }
 
+  /**
+   * Tests copying a file with attributes preserved.
+   */
+  @Test
+  public void copyFileWithPreservedAttributes() throws Exception {
+    InstancedConfiguration conf = new InstancedConfiguration(ServerConfiguration.global());
+    conf.set(PropertyKey.USER_FILE_WRITE_TYPE_DEFAULT, "MUST_CACHE");
```

Comment:
why do we need MUST_CACHE?

Reply:
Because persisted file will fail to change owner unless the test is ran with root account.

Reply:
ahhh, makes sense. Can we add a comment for this?

Reply:
Added a comment.